sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFsRelation.scala

```diff
@@ -19,11 +19,13 @@ package org.apache.spark.sql.execution.datasources
 
 import java.util.Locale
```

Comment:
This is irrelevant to this PR. Could you remove this unused import at this PR?

Reply:
Removed the unused import